airbyte/assets/monitors/long_running_jobs.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "Airbyte: Too Many Long-Running Jobs",
+	"type": "query alert",
+	"query": "avg(last_5m):avg:airbyte.metrics_reporter.num_unusually_long_syncs{*} / avg:airbyte.metrics_reporter.num_running_jobs{*} * 100 > 5",
+	"message": "{{#is_alert}}\n{{value}}% jobs are taking too long to complete which is higher than the threshold of {{ok_threshold}}%.\n{{/is_alert}} \n\n{{#is_recovery}}\nThe share of long-running jobs is back to normal!\n{{/is_recovery}}",
+	"tags": [
+		"integration:airbyte"
+	],
+	"options": {
+		"thresholds": {
+			"critical": 5
```

Comment:
Non blocking question, so if 5% of job are long running, the monitor will trigger?

Reply:
That's exactly right!